### PR TITLE
optimize category endpoints are service-specific

### PR DESCRIPTION
**Pull request by MartinRinas**

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/office-365-vpn-split-tunnel.md [hunk 1]
```diff
@@ -73,6 +73,7 @@ For the Office 365 service, Microsoft has designed the connectivity requirements
 Office 365 categorizes the required endpoints for Office 365 into three categories: **Optimize**, **Allow** and **Default**. **Optimize** endpoints are our focus here and have the following characteristics:
 
 - Are Microsoft owned and managed endpoints, hosted on Microsoft infrastructure
+- Are dedicated to core Office 365 workloads such as Exchange Online, SharePoint Online, Skype for Business Online and Microsoft Teams
```
Comment (JohanFreelancer9) [inline Enterprise/office-365-vpn-split-tunnel.md:76]:
```suggestion
- Are dedicated to core Office 365 workloads such as Exchange Online, SharePoint Online, Skype for Business Online, and Microsoft Teams
```

File: Enterprise/office-365-vpn-split-tunnel.md [hunk 2]
```diff
@@ -73,6 +73,7 @@ For the Office 365 service, Microsoft has designed the connectivity requirements
 Office 365 categorizes the required endpoints for Office 365 into three categories: **Optimize**, **Allow** and **Default**. **Optimize** endpoints are our focus here and have the following characteristics:
```
Comment (JohanFreelancer9) [inline Enterprise/office-365-vpn-split-tunnel.md:73]:
```suggestion
Office 365 categorizes the required endpoints for Office 365 into three categories: **Optimize**, **Allow**, and **Default**. **Optimize** endpoints are our focus here and have the following characteristics:
```

File: Enterprise/office-365-vpn-split-tunnel.md [hunk 3]
```diff
@@ -73,6 +73,7 @@ For the Office 365 service, Microsoft has designed the connectivity requirements
...
 
 - Are Microsoft owned and managed endpoints, hosted on Microsoft infrastructure
+- Are dedicated to core Office 365 workloads such as Exchange Online, SharePoint Online, Skype for Business Online and Microsoft Teams
 - Have IPs provided
 - Low rate of change and are expected to remain small in number(currently 20 IP subnets)
```
Comment (JohanFreelancer9) [inline Enterprise/office-365-vpn-split-tunnel.md:78]:
```suggestion
- Low rate of change and are expected to remain small in number (currently 20 IP subnets)
```

**Comment by mypil** [comment]
@MartinRinas - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by mypil** [comment]
@kelleyvice-msft - PR has been copy edited and is ready for final review.  This may be based on issue #802.

Thank you.

cc: @kenwith

**Pull request merged**